### PR TITLE
Adding k8 1.26 support for PowerMax

### PR DESCRIPTION
**Pull request by boyamurthy**
Adding k8 1.26 support for PowerMax

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/597|

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/powermax.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ CRDs should be configured during replication prepare stage with repctl as descri
 - This script also runs the verify.sh script in the same directory. You will be prompted to enter the credentials for each of the Kubernetes nodes. The `verify.sh` script needs the credentials to check if the iSCSI initiators have been configured on all nodes. You can also skip the verification step by specifying the `--skip-verify-node` option
 - In order to enable authorization, there should be an authorization proxy server already installed. 
 - PowerMax Array username must have role as `StorageAdmin` to be able to perform CRUD operations.
-- If the user is using complex K8s version like “v1.22.3-mirantis-1”, use below kubeVersion check in [helm Chart](https://github.com/dell/csi-powermax/blob/main/helm/csi-powermax/Chart.yaml) file. kubeVersion: “>= 1.22.0-0 < 1.25.0-0”.
+- If the user is using complex K8s version like “v1.23.3-mirantis-1”, use below kubeVersion check in [helm Chart](https://github.com/dell/csi-powermax/blob/main/helm/csi-powermax/Chart.yaml) file. kubeVersion: “>= 1.23.0-0 < 1.27.0-0”.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/powermax.md:296]: Change "use below" to "use this"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Two minor edits are required.

File: content/docs/csidriver/installation/helm/powermax.md (hunk 1 above)
Comment (boyamurthy) [reply content/docs/csidriver/installation/helm/powermax.md]: resolved.

File: content/docs/csidriver/troubleshooting/powermax.md [hunk 2]
```diff
@@ -9,6 +9,6 @@ description: Troubleshooting PowerMax Driver
...
+|Driver install or upgrade fails because of an incompatible Kubernetes version, even though the version seems to be within the range of compatibility. For example: Error: UPGRADE FAILED: chart requires kubeVersion: >= 1.23.0 < 1.27.0 which is incompatible with Kubernetes V1.23.11-mirantis-1 | If you are using an extended Kubernetes version, please see the [helm Chart](https://github.com/dell/csi-powermax/blob/main/helm/csi-powermax/Chart.yaml) and use the alternate kubeVersion check that is provided in the comments. Please note that this is not meant to be used to enable the use of pre-release alpha and beta versions, which is not supported.|
```
[the remark this answers is not in the document]
Comment (boyamurthy) [reply content/docs/csidriver/troubleshooting/powermax.md]: updated .

**Comment by boyamurthy** [comment]
> I have reviewed this. Two minor edits are required.

Thanks @rsedlock1958  for the review and have updated the comments .

**Review by delldubey — APPROVED** [review]
lgtm

**Review by shanmydell — APPROVED** [review]

**Pull request merged**